.github/workflows/push-docker-image.yml

```diff
@@ -0,0 +1,55 @@
+name: Push to Docker Hub
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - "*.*.*"
+  pull_request:
```

Comment:
AFAIU, the [pull_request](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request) event is triggered when the PR metainfo (comments/labels/assigned people) changes but is not triggered when new commits are pushed.

Is this the intended behavior?

Reply:
As you can see from the history of this PR, the event is indeed triggered on every push :) The [synchronize](https://github.community/t/what-is-a-pull-request-synchronize-event/14784/4) event is triggered on each push to the branch of PR

Reply:
Got it, thanks for the explanation!